security/RFP.md

```diff
@@ -0,0 +1,254 @@
+
+You will review the corresponding artifacts described above to assess that the Hydra node logic layer code faithfully implements the off-chain logic described in the Coordinated Hydra Head V1 specification. The outcome of the review should include, bo not being limited to:
+* identify any discrepancy with the off-chain logic from the specification;
+* for any such discrepancy, what security property could be impacted;
+* a review of the tests applied to the Hydra node logic layer and, in particular, any situation that would not be covered by the tests.
+
+This review should consider fair management of the input and output to and from the Hydra node logic layer.
+
+### Hydra head protocol implementation is immune to attacks via chain transactions
+
+The Hydra node chain layer is in charge of observing transactions posted on the Cardano blockchain
+and filtering _interesting_ transaction to submit them to the logic layer. The hydra node logic layer
+then analyse those transactions and react to them to make a head status evolve.
+
+You will review the Hydra node chain and logic layers and the corresponding artifacts in presence of
+invalid, unexpected or maliciously forged transactions. The outcome of the review should include, but not being limited to:
+* identification of transactions or a suite of transactions that would mislead a hydra node to an incorrect head state
+* identification of transactions or a suite of transactions that would lead to a denial of service for the hydra node
+* identification of transactions or a suite of transactions that would invalidate one of the security properties of the Coordinated Hydra Head V1 specification.
+
+### Hydra head protocol implementation is immune to attacks via network
+
+The Hydra node network layer is in charge of receiving peers messages and connections and sending network
+messages to peers. It decodes received messages andsubmit them to the logic layer. The hydra node logic layer
+then analyse those messages and react to them to make a head status evolve.
+
+You will review the Hydra node network and logic layers and the corresponding artifacts in presence of
+invalid, unexpected or maliciously forged network messages. The outcome of the review should include, but not being
+limited to:
+* identification of network attack that would mislead a hydra node to an incorrect head state
+* identification of network attack that would lead to a denial of service for the hydra node
+* identification of network attack that would invalidate one of the security properties of the Coordinated Hydra Head V1 specification.
+
+TODO: should we explicitly state that we are not interested in network denial of service attack? In the hypotheses, should we consider only that locally submited transaction will end up reaching the cardano network or should we enforce stronger network propoerties? For instance, if a node totally loose its network connection, is it OK? For how long is it OK?
```

Comment:
I think this task can be about network DoS. Are you considering asking for checking whether the hydra network connections can be used to DoS the whole machine, which would affect the network connection of the `cardano-node` on the same machine? If yes .. that is likely possible and a valid point for this task!